.travis.yml

```diff
@@ -7,19 +7,20 @@ env:
   - TOX_ENV=py27
   - TOX_ENV=py33
   - TOX_ENV=py34
+  - TOX_ENV=py35
 matrix:
-  include:
-    - python: 3.5
-      env:
-      - TOX_ENV=py35
+    include:
+        - python: 3.6
+          env:
+              - TOX_ENV=py36
```

Comment:
It would probably be better to keep indentation here consistent with the rest of the file (2 spaces).